src/maintainer/maintainer_faq.rst

```diff
@@ -110,4 +110,9 @@ This message can be safely ignored as conda-forge doesn't use conda-verify.
 
     The exit code 139 itself actually is the general exit code for a segmentation fault. This could also mean that you have run into a different issue but the above issue is the most likely one with our CentOS 6-based images.
 
+.. _mfaq_package_submit:
+
+:ref:`(Q) <mfaq_package_submit>` **Is it necessary for me to be a maintainer of the package I submit to Conda-forge?**
+	Everybody can submit a package to Conda-forge, irrespective of whether they maintain it or not. Additionally, it’s not required but considered a good practice to inform the upstream of a new package and invite them to be maintainers as well.
```

Comment:
```suggestion
	Everybody can submit a package to Conda-forge, irrespective of whether they maintain the upstream version or not. Additionally, it’s not required but considered good practice to inform the upstream of a new package and invite them to be maintainers as well.
```
(Or package instead of version?)

Reply:
nah, I think that's good